Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp

```diff
@@ -6,6 +6,9 @@
 #include <cstdio>
 #include <cstdlib>
 #include <string>
+#include <arpa/inet.h>
```

Comment:
Same here; this is breaking Windows builds, and it ends up including those headers twice. Perhaps change the `#if` below to add `__OpenBSD__` instead?